package NYT.NTableClient.NProto;

import "yt/ytlib/table_client/schema.proto";

////////////////////////////////////////////////////////////////////////////////

message TReadLimit
{
    optional int64 row_index = 1;
    optional TKey key = 2;
}

message TChunkSlice
{
    required bytes chunk_id = 1;
    required TReadLimit start_limit = 2;
    required TReadLimit end_limit = 3;
}

////////////////////////////////////////////////////////////////////////////////
